src/components/commons/MetascanLinkButton.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import classNames from "classnames";
+import { useTheme } from "@contexts/ThemeContext";
+
+interface ViewOnMetascanButtonProps {
+  href?: string;
+  customStyle?: string;
+}
+
+export function MetascanLinkButton({
+  href,
+  customStyle,
+}: ViewOnMetascanButtonProps): JSX.Element {
+  return (
+    <a
+      href={href}
+      target="_blank"
+      rel="noreferrer"
+      data-testid="metascan-link-button"
+    >
+      <button
+        type="button"
+        data-testid="view-on-metachain-button"
+        className={classNames(
+          "border-[0.5px] rounded-[30px] border-gray-200 dark:border-gray-700",
+          "flex items-center gap-2",
+          "py-1.5 px-2 text-sm font-medium text-gray-700 dark:text-gray-300 tracking-[0.0025em]",
+          customStyle,
+        )}
+      >
+        <DFILogo />
+        View on Metachain
```

Comment:
```suggestion
        View on MetaChain
```

Reply:
Aren't there any hover effect on the button? so it would feel like one - ux-wise? 🤔